src/_P183_modbus.ino

```diff
@@ -234,8 +236,9 @@ boolean Plugin_183(uint8_t function, struct EventStruct *event, String& string)
                                    P183_GET_FLAG_COLL_DETECT)) {
         break;
       }
+      addLogMove(LOG_LEVEL_INFO, "P183 INIT AFTER INIT");
       P183_ModbusDevice->setModbusTimeout(P183_MODBUS_TIMEOUT);
-
+      addLogMove(LOG_LEVEL_DEBUG, "AFTER TIMEOUT");
```

Comment:
Please check to see if we have debug code (including this LOG_LEVEL_DEBUG define) included in the build.
```c++
#ifndef BUILD_NO_DEBUG
addLogMove(LOG_LEVEL_DEBUG, F("AFTER TIMEOUT"));
#endif
```

And also please use the F() macro as I showed here when handling strings.